plugins/tiddlywiki/markdown/settings.tid

```diff
@@ -0,0 +1,19 @@
+title: $:/plugins/tiddlywiki/markdown/settings
```

Comment:
If we also add "settings" to the list field of the `plugin.info` file then the settings will also show up as a tab within the view of the plugin, which I think is useful.